app/src/main/res/layout/toolbar_basic.xml

```diff
@@ -6,14 +6,14 @@
         android:id="@+id/appbar"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
+        android:background="?android:colorBackground"
         android:elevation="@dimen/actionbar_elevation"
```

Comment:
In the Studio I see the change
`app:elevation="@dimen/actionbar_elevation"` -> `android:elevation="@dimen/actionbar_elevation"`
but not here and I'm confused.